app/Resources/views/committee/members.html.twig

```diff
@@ -6,6 +6,44 @@
 
 {% block banner '' %}
 
+{% block javascripts %}
+    <script type="application/javascript">
+        var allCheckbox = dom('#members-check-all');
```

Comment:
Nevermind... It targets the checkbox in the header section. I thought it meant "all checkboxes" elements.